backends/gdx-backend-robovm/src/com/badlogic/gdx/backends/iosrobovm/IOSGraphics.java

```diff
@@ -59,7 +59,7 @@
 public class IOSGraphics extends NSObject implements Graphics, GLKViewDelegate, GLKViewControllerDelegate {
 
 	private static final String tag = "IOSGraphics";
-
+	volatile boolean resume = false;

```

Comment:
We can move it with the rest of variable declarations.

I wonder if we need it `volatile`. I'm not sure `resume()` or `draw()` can be called from any other than the main thread. I guess it's not a bad idea to be on the safe side do the same as we do with `paused`.

Reply:
It's been moved but it was kept volatile